README.md

```diff
@@ -94,6 +94,7 @@ When *Boost.Fiber* is enabled and alpaka is build in C++17 mode with clang and l
 When an accelerator back-end using *CUDA* is enabled, version *8.0* of the *CUDA SDK* is the minimum requirement.
 *NOTE*: When using nvcc as *CUDA* compiler, the *CUDA accelerator back-end* can not be enabled together with the *Boost.Fiber accelerator back-end* due to bugs in the nvcc compiler.
```

Comment:
Note on the `*Note*:` lines ;-)

If you add *two spaces* at the end of a sentence, this will cause a line-break for each note but not a new paragraph :) This might make this more readable here.

Reply:
Thanks for the tip. Looks much better now.